c10/core/SymBool.h

```diff
@@ -52,6 +52,7 @@ class C10_API SymBool {
   // that value.  Note that C++ comparison operations default to returning
   // bool, so it's not so common to have to call this
   bool guard_bool(const char* file, int64_t line) const;
+  bool expect_true(const char* file, int64_t line) const;
```

Comment:
guard_true feels more in line with (1) guard_bool (2) this not being a test (expect... feels firmly like testing nomenclature)

Reply:
It's not a guard. They do strictly different things.

The expect nomenclature is inline with `__builtin_expect` which matches what is going on here exactly.